bitnami/scylladb/6.2/debian-12/rootfs/opt/bitnami/scripts/libscylladb.sh

```diff
@@ -797,8 +797,8 @@ cassandra_setup_common_ports() {
 #########################
 cassandra_setup_rack_dc() {
     if ! cassandra_is_file_external "${DB_MOUNTED_RACKDC_PATH}"; then
-        replace_in_file "${DB_RACKDC_FILE}" "dc=.*" "dc=${DB_DATACENTER}"
-        replace_in_file "${DB_RACKDC_FILE}" "rack=.*" "rack=${DB_RACK}"
+        replace_in_file "${DB_RACKDC_FILE}" "^(#\s)?(\s*)dc=.*" "dc=${DB_DATACENTER}"
+        replace_in_file "${DB_RACKDC_FILE}" "^(#\s)?(\s*)rack=.*" "rack=${DB_RACK}"
```

Comment:
What about simplifying the regex? 

```suggestion
        replace_in_file "${DB_RACKDC_FILE}" "^[#\s]*dc=.*" "dc=${DB_DATACENTER}"
        replace_in_file "${DB_RACKDC_FILE}" "^[#\s]*rack=.*" "rack=${DB_RACK}"
```

Reply:
Looks good to me 🙂  I copied the expression from other functions in the script.

Updated the patch with your proposed change.